packages/frontend/web/pages/Article.tsx

```diff
@@ -89,14 +89,20 @@ const headerAd = css`
 // These are by selector as for dynamically-created ads
 const bodyAdStyles = css`
     .ad-slot {
-        background-color: ${palette.neutral[97]};
-        width: 320px;
+        width: 300px;
```

Comment:
Discussed in person - this being the default width, and wide ads should override it in the body.